PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -606,6 +661,10 @@ DECLARE_SOA_TABLE(HfCand2ProngBase, "AOD", "HFCAND2PBASE", //!
                   hf_cand::ErrorImpactParameterZ0, hf_cand::ErrorImpactParameterZ1,
                   hf_track_index::Prong0Id, hf_track_index::Prong1Id, hf_cand::NProngsContributorsPV,
                   hf_track_index::HFflag,
+                  hf_cand::NSigTpcPi0, hf_cand::NSigTpcKa0,
+                  hf_cand::NSigTpcPi1, hf_cand::NSigTpcKa1,
+                  hf_cand::NSigTofPi0, hf_cand::NSigTofKa0,
+                  hf_cand::NSigTofPi1, hf_cand::NSigTofKa1,
```

Comment:
I would put these in a separated joinable table, to allow the users to decide whether to consume it or not (for instance needed to be consumed in the selector, but not necessarily later on)